packages/commerce-sdk-react/src/auth/storage/cookie.ts

```diff
@@ -31,13 +31,45 @@ export class CookieStorage extends BaseStorage {
     }
     get(key: string) {
         const suffixedKey = this.getSuffixedKey(key)
-        return Cookies.get(suffixedKey) || ''
+        let value = Cookies.get(suffixedKey) || ''
+
+        // TODO: How can we automate a test for this?
```

Comment:
Can you please remove this. A better approach would be to create new ticket to add a test for this.